### PR TITLE
Issue #4686: Modified CheckstyleAntTaskTest.java and moved its input files to the 'checkstyleanttask' subdirectory

### PR DESCRIPTION
**Pull request by subkrish**
Issue #4686 

This PR moves all inputs of `CheckstyleAntTaskTest` of the `'ant'` package to a new subdirectory `'checkstyleanttask'`.

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java [hunk 1]
```diff
@@ -79,9 +79,10 @@
         "ant/checkstyleanttask/InputCheckstyleAntTaskError.java";
     private static final String WARNING_INPUT =
         "ant/checkstyleanttask/InputCheckstyleAntTaskWarning.java";
-    private static final String CONFIG_FILE = "ant/ant_task_test_checks.xml";
+    private static final String CONFIG_FILE = "ant/checkstyleanttask/ant_task_test_checks.xml";
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Can we make these all `InputCheckstyleAntTask...`?

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java [hunk 2]
```diff
@@ -79,9 +79,10 @@
...
     private static final String CONFIG_RESOURCE = "/com/puppycrawl/tools/checkstyle/" + CONFIG_FILE;
-    private static final String CUSTOM_ROOT_CONFIG_FILE = "config-custom-root-module.xml";
+    private static final String CUSTOM_ROOT_CONFIG_FILE =
+        "ant/checkstyleanttask/config-custom-root-module.xml";
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Same.

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java [hunk 3]
```diff
@@ -476,7 +477,8 @@ public final void testSetPropertiesFile() throws IOException {
 
         final CheckstyleAntTask antTask = getCheckstyleAntTask(CUSTOM_ROOT_CONFIG_FILE);
         antTask.setFile(new File(getPath(VIOLATED_INPUT)));
-        antTask.setProperties(new File(getPath("ant/checkstyleAntTest.properties")));
+        antTask.setProperties(new File(getPath("ant/checkstyleanttask/"
+                + "checkstyleAntTest.properties")));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Same.

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java [hunk 4]
```diff
@@ -515,7 +517,7 @@ public final void testXmlOutput() throws IOException {
         antTask.execute();
 
         final List<String> expected = FileUtils.readLines(
-                new File(getPath("ant/ant_task_xml_output.xml")));
+                new File(getPath("ant/checkstyleanttask/ant_task_xml_output.xml")));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Same.

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java (hunk 1 above)
Comment (subkrish) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java (hunk 2 above)
Comment (subkrish) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java (hunk 3 above)
Comment (subkrish) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java (hunk 4 above)
Comment (subkrish) [inline src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java]: Done.

**Comment by rnveach** [comment]
@subkrish What about line 731?
`getCheckstyleAntTask("config-custom-checker-root-module.xml");`

**Comment by subkrish** [comment]
@rnveach I added that file as well `config-custom-checker-root-module.xml` -> `InputCheckstyleAntTaskConfigCustomCheckerRootModule.xml`. Done.

**Comment by subkrish** [comment]
@romani @rnveach Please see https://github.com/checkstyle/checkstyle/issues/4686#issuecomment-314007275

**Comment by rnveach** [comment]
@romani issue is ATLEAST to organize inputs into 1 folder. I don't mind if it isn't `checkstyleanttask` and instead `ant`.

**Comment by romani** [comment]
ok, lets merge this as some step in right direction.

**Pull request merged**